src/raftstore/store/store.rs

```diff
@@ -1838,49 +1853,148 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         epoch: metapb::RegionEpoch,
         split_key: Vec<u8>,
     ) {
+        self.on_prepare_split_region(
+            keys::origin_key(split_key.as_slice()).to_vec(),
+            Some(region_id),
+            Some(epoch),
+            None,
+        );
+    }
+
+    fn on_prepare_split_region(
+        &mut self,
+        split_key: Vec<u8>, // `split_key` is a encoded key.
+        region_id: Option<u64>,
```

Comment:
We must know the region id here. Don't allow the user send a request without a region id.

Reply:
E.g, if the user creates a table and wants to split the region, he/she must find the correct the region, then sends the split request.